n/neo4j/neo4j-5.9.0-ubi8.7.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#---------------------------------------------------------------------------------------------------
+#
+# Package		: Neo4j
+# Version		: 5.9.0
+# Source repo		: https://github.com/neo4j/neo4j.git
+# Tested on		: UBI 8.7 (docker)
+# Language		: Java
+# Travis-Check		: true
+# Script License	: Apache License, Version 2 or later
+# Maintainer		: Sumit Dubey <sumit.dubey2@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#---------------------------------------------------------------------------------------------------
+
+set -eux
+
+CWD=$(pwd)
+PACKAGE_NAME=neo4j
+PACKAGE_URL=https://github.com/neo4j/neo4j.git
+PACKAGE_VERSION=5.9.0
+MAVEN_VERSION=3.8.8
+GOSU_VERSION=1.16
+
+#Install RHEL deps
+yum install java-17-openjdk-devel git wget curl hostname procps-ng -y \
+    && curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/${GOSU_VERSION}/gosu-ppc64el" \
```

Comment:
do we need to install gosu? I don't see it referenced directly anywhere but is it a build dependency?

Reply:
yes the build needs it